### DISCUSSION TITLE
Where should I supply Degoo specific username and password?

### DISCUSSION THREAD

**Discussion by kraghuprasad** [bernd-wechner/Degoo#43] [Q&A]
I am not sure if I am the only one who could not figure it out. I did not find a single configuration file with place holders were user-id and password of Degoo service could be supplied. In the code it seems there is a configuration directory named degoo/ which is expected to have some json files. But I did not find any samples there. Is there some kind of documentation from where required details can be figured out?

**Comment by tomaasz** [comment]
It would be helpful to see tutorial on YouTube
